docs/flashbots-mev-boost/glossary.md

```diff
@@ -3,26 +3,25 @@ title: Glossary
 ---
 
 
-- **execution payload**: a [message](https://github.com/ethereum/consensus-specs/blob/a45ee9bf5b1fde766d69e551a6b1a21fe2531734/specs/merge/beacon-chain.md#executionpayload) containing the complete content of an unsigned execution payload (an object containing block properties in addition to transactions) that is provided by the execution layer (previous PoW nodes).
```

Comment:
Were these things just shifted in order? Or did some of the content change? (Why the re-order)?

Reply:
Changed to alphabetical order to make it easier to use.

Reply:
Ack!